app/app.js

```diff
@@ -1,14 +1,23 @@
-global.$ = window.$ = global.jQuery = window.jQuery = require('jquery');
-import 'bootstrap'
-import './app.sass'
-import 'babel-polyfill'
+global.$ = window.$ = global.jQuery = window.jQuery = require("jquery");
+import "bootstrap";
```

Comment:
(thinking about soon to come eslint config here)

Do we have a convention for single vs double quotes?  I see a mix throughout the code base so wanted to call out.  If we do we should stick with that.

If we don't we should decide :)

Reply:
We do need to decide. I'm using the default Prettier double quotes, however the codebase uses a CoffeeScript best practice which is single quotes unless using interpolation.

According to this [resource](https://bytearcher.com/articles/single-or-double-quotes-strings-javascript/) we should do single quotes to fit in with other code bases.